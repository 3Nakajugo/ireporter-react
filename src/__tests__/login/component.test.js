import React from 'react';
import { shallow } from 'enzyme';
import LoginComponent from '../../components/Login/login'


describe('<LoginComponent />', () => {
  it('should render login component', () => {
    const wrapper = shallow(<LoginComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});