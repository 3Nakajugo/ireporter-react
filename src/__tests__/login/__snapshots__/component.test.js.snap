// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginComponent /> should render login component 1`] = `ShallowWrapper {}`;
